config/common.php

```diff
@@ -63,6 +63,9 @@
             'enableAutoLogin' => true,
             'loginUrl' => ['/user/auth/login'],
             'identityClass' => 'modules\user\common\components\UserIdentity'
+        ],
+        'assetManager' => [
+            'linkAssets' => YII_ENV_DEV ? true : false
```

Comment:
@samirreza this line is needed because..? 
unfortunately, it makes problem in some windows environments.